### ISSUE TITLE
FastSEResNet and DUC layer

### ISSUE THREAD

**Issue by 7oud** [MVIG-SJTU/AlphaPose#470]
I found AlphaPose is totally different from RMPE, it is faster and more accurate but without the key features of SSTN, Parallel SPPE and etc. 
AlphaPose is simpler and the SPPE is use FastSEResNet and DUC layer, could you give the  papers where these layers come from? and why they work well?

**Comment by Fang-Haoshu** [comment]
Hi, thanks for your interest. In fact we are about to write the paper, but not yet 😂

**Comment by 7oud** [comment]
@Fang-Haoshu I try to train the model using [gluoncv code](https://github.com/dmlc/gluon-cv/tree/master/scripts/pose/alpha_pose) with 1 gpu, but the network didn't converge and val accuracy is quite low, Is the gluoncv version same with this mxnet branch?
I found the training data was a little different from gluoncv's, 
- trainval2017 is train2017 + val2017?
- annot_coco.h5 is the same with annot.h5 in the master branch?
- why use person_keypoints_val2014.json not 2017?

**Comment by Fang-Haoshu** [comment]
The gluoncv code should work properly. We have worked for a long time on that PR. It's a bit different.

**Comment by gusphak** [comment]
> Hi, thanks for your interest. In fact we are about to write the paper, but not yet 😂

Just wanted to know whether the paper had been published yet.

**Comment by dekathomas** [comment]
Hello @7oud, did you said that AlphaPose doesn't use SSTN, Parallel SPPE, etc. Do you know the full pipeline of this AlphaPose? What technique does it use?

**Comment by Fang-Haoshu** [comment]
Hi, the paper is public at http://arxiv.org/abs/2211.03375
Thanks for your interest!
